app/consumers.py

```diff
@@ -31,15 +31,16 @@ async def disconnect(self, close_code):
             self.room_group_name,
             self.channel_name          
         )
-        print('consumer disconnect')        
+        print('consumer disconnect')
+        print(close_code)
         
-        # Send message to room group
+        # # Send message to room group
```

Comment:
Why were lines 37-45 not deleted?

Reply:
placeholder for possible feature in future